@xen-orchestra/web-core/lib/components/state-hero/VtsLoadingHero.vue

```diff
@@ -1,13 +1,51 @@
 <template>
-  <VtsStateHero :type busy>
-    {{ $t('loading-in-progress') }}
-  </VtsStateHero>
+  <div class="vts-loading-hero">
+    <VtsStateHero :type busy>
+      {{ $t('loading-in-progress') }}
+    </VtsStateHero>
+    <div class="content">
+      <div v-if="slots.title" class="title" :class="typoClass">
+        <slot name="title" />
+      </div>
+      <div v-if="slots.text" class="text typo-body-bold">
+        <slot name="text" />
+      </div>
+    </div>
+  </div>
 </template>
 
 <script lang="ts" setup>
 import VtsStateHero, { type StateHeroType } from '@core/components/state-hero/VtsStateHero.vue'
+import { computed } from 'vue'
 
-defineProps<{
+const props = defineProps<{
```

Comment:
I think 90% of the time, the title and subtitle will be only text, so maybe you can use this [pattern](https://github.com/vatesfr/xen-orchestra/blob/master/%40xen-orchestra/web-core/docs/guidelines/best-practices.md#use-slot--prop-for-better-flexibility).